openshift-knative-operator/cmd/operator/kodata/knative-serving/0.17.3/5-serving-post-install-jobs.yaml

```diff
@@ -18,7 +18,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: storage-version-migration-serving-
+  name: storage-version-migration-serving-0.17.3
```

Comment:
:stuck_out_tongue_closed_eyes:

Reply:
Question.

if we would use our forks as dependency, we could patch on the forks ? instead of touching the vendored? 

Not sure if all the hassle is really worth it

Reply:
Nope. Good point actually, I shall send another PR before this: We need to actually create these manifests ourselves. They don't appear here by magic. I'll add download scripts that do that for us. We can then add automatic patches into those scripts too, if really necessary.

Reply:
For eventing, we create some yamls already:
https://github.com/openshift/knative-eventing/blob/release-v0.17.2/openshift/release/knative-eventing-ci.yaml#L3077

but we are not really touching the jobs yet - I guess worth to do it?

Reply:
Download scripts are in place meanwhile.